internal/listener/listener.go

```diff
@@ -158,14 +163,18 @@ func (l *Listener) ProcessEvent(w http.ResponseWriter, req *http.Request) {
 func (l *Listener) checkDebugPassword(w http.ResponseWriter, r *http.Request) bool {
 	expectedPassword := l.configFile.DebugPassword
 	if expectedPassword == "" {
+		l.logger.Warnw("Config dump disables, no debug-password set in config", zap.String("url", r.RequestURI), zap.String("agent", r.UserAgent()))
+
```

Comment:
I wouldn't log this here as there's little point seeing this in the logs. After all, these endpoints are intended for manual consumption, so you'll see that message (not like in some source doing whatever it does with the response body).